src/client/lib/_mapbox/baselayer-control.js

```diff
@@ -0,0 +1,20 @@
+export default class BaseLayerControl {
```

Comment:
I'd expect this to live closer to `components/baselayer-control`

Reply:
It's not a Vue component it's specific to mapbox thats why I put it here